charts/spire/charts/spire-agent/values.yaml

```diff
@@ -94,6 +94,22 @@ waitForIt:
     tag: latest-20230517
   resources: {}
 
+bash:
+  image:
+    # -- The OCI registry to pull the image from
+    registry: cgr.dev
+    # -- The repository within the registry
+    repository: chainguard/bash
+    # -- The image pull policy
+    pullPolicy: Always
+    # -- This value is deprecated in favor of tag. (Will be removed in a future release)
+    version: ""
+    # -- Overrides the image tag
+    tag: latest-20230517
+
+fsGroupFix:
+  resources: {}
```

Comment:
What is this used for?

Reply:
For users to specify the resources needed by the container. A best practice on production clusters. https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

Reply:
Can you add a comment for this?

Reply:
We don't currently document it anywhere else, but gotta start somewhere I guess.

Reply:
done